src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,14 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        int ballCount = 3;
+        Ball[] ballArray = new Ball[ballCount];
+        for (int i = 0; i < ballCount; i++) {
+            ballArray[i] = lottery.getRandomBall();
+        }
+        for (Ball ball : ballArray) {
+            System.out.println(ball);
+        }
```

Comment:
Can be simplified to one loop
```suggestion
            System.out.println(lottery.getRandomBall());
        }
```